packages/zowe-explorer/__tests__/__unit__/Profiles.extended.unit.test.ts

```diff
@@ -1665,7 +1665,12 @@ describe("Profiles Unit Tests - function ssoLogout", () => {
             globalMocks.testSession,
             globalMocks.testProfile
         );
-        testNode.profile.profile.password = undefined;
+        // testNode.profile.profile.password = undefined;
+        delete testNode.profile.profile.password;
+        // Note: The above needs to be deleted instead of set to undefined.
+        //      Otherwise the `ProfilesUtils.isProfileUsingBasicAuth` reports that we are using basic auth
+        //      because "password" in testNode.profile.profile is "true" even though it's undefined
```

Comment:
Is the check in `ProfilesUtils.isProfileUsingBasicAuth` behaving as expected? Curious if this:
`"user" in prof && "password" in prof;`
should be changed to:
`prof.user != null && prof.password != null;`

Reply:
Opened: 
- https://github.com/zowe/vscode-extension-for-zowe/issues/2664